integ-tests/src/main/java/com/google/tdjdbc/JdbcHelper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.tdjdbc;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public final class JdbcHelper {
+
+    public static String getStringNotNull(ResultSet rs, String column) throws SQLException {
+        String buffer = rs.getString(column);
+        if (rs.wasNull()) {
```

Comment:
Alternatively, you can use Guava's [`firstNonNull`](https://guava.dev/releases/30.1-jre/api/docs/com/google/common/base/MoreObjects.html#firstNonNull(T,T)) (with static import):

```java
return firstNonNull(rs.getString(column), "");
```

Reply:
Hmm, do you want me to change it or it's just an optional recommendation?

Reply:
I meant it as optional, but I now realize that my suggestion does not work anyway because that isn't a normal null check but using `wasNull()`.